corehq/apps/cloudcare/static/cloudcare/js/formplayer/apps/views.js

```diff
@@ -112,6 +112,13 @@ hqDefine("cloudcare/js/formplayer/apps/views", [
         restoreAsKeyAction: _.extend(BaseAppView.restoreAsKeyAction),
         settingsKeyAction: _.extend(BaseAppView.settingsKeyAction),
 
+        // Cannot append at the end of the parent because there are
+        // special grid items.
+        attachHtml: function attachHtml(els, $container) {
+            let childElement = $container.find("#put-apps-here");
+            $container[0].insertBefore(els, childElement[0]);
+        },
```

Comment:
I don't love this solution but I looked at it for a while and couldn't come up with anything better.

Reply:
same here